README.md

```diff
@@ -2162,6 +2163,74 @@ export function reply_raw(): UpdateManual<RawReply> {
 
 [Not yet implemented.](https://github.com/demergent-labs/azle/issues/496)
 
+### Timer APIs
+
+-   [clear timer](#clear-timer)
+-   [set timer](#set-timer)
+-   [set timer interval](#set-timer-interval)
+
+#### clear timer
+
+Examples:
+
+-   [timers](/examples/timers)
+
```

Comment:
Can you add a more information section like elsewhere in the documentation and link to the timers documentation on internetcomputer.org? If you could do this for all of the APIs that would be great

Reply:
The documentation on this hasn't been published by Dfinity yet. It's available at https://github.com/dfinity/interface-spec/blob/master/spec/index.adoc#global-timer, but let's wait to reference it till it's on https://internetcomputer.org/docs/current/references/ic-interface-spec.